vraptor-javatime/src/main/java/br/com/caelum/vraptor/converter/InstantConverter.java

```diff
@@ -0,0 +1,44 @@
+package br.com.caelum.vraptor.converter;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+import java.time.Instant;
+import java.time.format.DateTimeParseException;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+
+import br.com.caelum.vraptor.Convert;
+
+@RequestScoped
+@Convert(Instant.class)
+public class InstantConverter implements Converter<Instant> {
+	private final Logger log;
+	
+	/** @deprecated CDI */ @Deprecated
+	InstantConverter() { this(null); }

```

Comment:
could you update this using the same convention of the other converters?

https://github.com/caelum/vraptor-time-converters/blob/master/vraptor-javatime/src/main/java/br/com/caelum/vraptor/converter/LocalDateConverter.java#L45-L50